administration/src/project-configs/getProjectConfig.ts

```diff
@@ -114,6 +114,7 @@ export interface ProjectConfig {
   freinetCSVImportEnabled: boolean
   cardCreation: boolean
   storeManagement: StoresManagement
+  userUploadApiEnabled: boolean
```

Comment:
```suggestion
  userImportApiEnabled: boolean
```
just to stick to the same naming